cmd/ttn-lw-cli/commands/gateways.go

```diff
@@ -394,3 +426,17 @@ func init() {
 	gatewaysCommand.AddCommand(gatewaysContactInfoCommand)
 	Root.AddCommand(gatewaysCommand)
 }
+
+var errAddressMismatchGateway = errors.DefineAborted("gs_address_mismatch", "gateway server address mismatch")
```

Comment:
Error ID should be `gateway_server_address_mismatch` for consistency with end devices.

Reply:
corrected